src/ui/public/chrome/api/__tests__/xsrf.js

```diff
@@ -114,7 +113,7 @@ describe('chrome xsrf apis', function () {
       });
 
       it('accepts alternate tokens to use', function () {
```

Comment:
Ah, sorry, this is still a thing, but it probably shouldn't be. If the user sends a custom value it will be used as the kbn-version header. This I'll change this value to be boolean-y